views/error.html

```diff
@@ -0,0 +1,23 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Sorry, there is a problem with the service - GOV.UK
+{% endblock %}
+
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back",
```

Comment:
Link takes you back to the beginning / home screen rather than the previous screen so you might want to check that the text displayed for this link is correct (not like on all the other screens where the link does take you back).

Reply:
After Talking to Oceanne as it is a system/service error and not due to Customer error it would be best to remove the back link entirely. This is also done in other services such as apply-to-protect.